k8-util/helm/fluvio-app/templates/sc-deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         {{- toYaml .Values.scPod.nodeSelector | nindent 8 }}
       containers:
         - name: fluvio-sc
-          image: {{ .Values.image.registry }}/fluvio:{{ .Values.image.tag | default .Chart.AppVersion }}
+          image: "{{ .Values.image.registry }}/fluvio:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
why quote?

Reply:
`helm` errors out without quotes:
```
Error: YAML parse error on fluvio-app/templates/sc-deployment.yaml: error converting YAML to JSON: yaml: line 22: mapping values are not allowed in this context
```
I was using `make install_minikube` in this case, tried version 3.3.4 and the latest 3.7.1 with the same result